add operation and settings (#905)
vesoft-inc/nebula-docs: docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md

# Operation record

> This topic will describe the operation record in Nebula Dashboard.

Reviewer: the refers to specified nouns. the operation record -->operation records

Reviewer: [describe the operation record] sounds weird. -->This topic shows how to use the operation record feature in xxx.

![operation record](../figs/ds-018.png)

> On the **Operation Record** page, you can check the operation record of the latest 1 hour, 6 hours, 1 day, 3 days, 7days or 14 days, including operator， operation， time and cluster name.

Reviewer: record -->records

Reviewer: [including operator， operation， time and cluster name.] sounds odd.

Reviewer 2: ```suggestion
On the **Operation record** page, you can check the operation record of the latest 1 hour, 6 hours, 1 day, 3 days, 7days, or 14 days. You can also view who runs what operation on which cluster at what time.
```
